contracts/Paper.sol

```diff
@@ -1,27 +1,20 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.2;
 
-import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
-import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Snapshot.sol";
+import "./ERC20Snapshot.sol";
 import "@openzeppelin/contracts/token/ERC721/extensions/IERC721Enumerable.sol";
-import "@openzeppelin/contracts/token/ERC20/extensions/draft-ERC20Permit.sol";
-import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 
-contract Paper is ERC20, ERC20Permit, ERC20Votes, ERC20Snapshot, Ownable {
+contract Paper is ERC20Snapshot, Ownable {
```

Comment:
we cant modify this source now since it is deployed and we need to match the deployed source code